logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -463,14 +463,20 @@ class CallManagerImpl internal constructor(
         callClients: CallClientList
     ) {
         withCalling {
-            // Needed to support calls between federated and non federated environments
+            // Mapping Needed to support calls between federated and non federated environments (domain separation)
             val clients = callClients.clients.map { callClient ->
                 CallClient(
-                    federatedIdMapper.parseToFederatedId(callClient.userId),
-                    callClient.clientId
+                    userId = federatedIdMapper.parseToFederatedId(callClient.userId),
+                    clientId = callClient.clientId,
+                    isMemberOfSubconversation = callClient.isMemberOfSubconversation,
+                    quality = callClient.quality
                 )
             }
             val clientsJson = CallClientList(clients).toJsonString()
+            callingLogger.d(
+                "$TAG - wcall_request_video_streams() called -> Requesting video streams for conversation = " +
+                        "${conversationId.toLogString()} with clients = $clientsJson"
```

Comment:
what clientsJson include in this case? and is it safe to log ?

Reply:
Removed, thanks for the catch